CARTO: QuadbinHeatmapTileLayer docs (#8909)
visgl/deck.gl: docs/api-reference/carto/quadbin-heatmap-tile-layer.md

# QuadbinHeatmapTileLayer (Experimental)

> `QuadbinHeatmapTileLayer` is a layer for visualizing tiled data described using the [Quadbin Spatial Index](https://docs.carto.com/data-and-analysis/analytics-toolbox-for-bigquery/key-concepts/spatial-indexes#quadbin) using a heatmap. 

Reviewer: Minor wording suggestion, to avoid repetition of "using".

```suggestion
`QuadbinHeatmapTileLayer` is a layer for visualizing tiled data indexed with a [Quadbin Spatial Index](https://docs.carto.com/data-and-analysis/analytics-toolbox-for-bigquery/key-concepts/spatial-indexes#quadbin) using a heatmap. 
```

<script src="https://unpkg.com/@deck.gl/carto@^9.0.0/dist.min.js"></script>
```

```js
new deck.carto.QuadbinHeatmapTileLayer({});
```

## Properties

Inherits from all [Base Layer](../core/layer.md) and [CompositeLayer](../core/composite-layer.md) properties.

#### `data` (TilejsonResult) {#data}

Required. A valid `TilejsonResult` object.

Use one of the following [Data Sources](./data-sources.md) to fetch this from the CARTO API:

- [quadbinTableSource](./data-sources#quadbintablesource)
- [quadbinQuerySource](./data-sources#quadbinquerysource)
- [quadbinTilesetSource](./data-sources#quadbintilesetsource)

### Render Options

#### `radiusPixels` (number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square") {#radiuspixels}

* Default: `20`

Radius of the heatmap blur in pixels, to which the weight of a cell is distributed.

#### `colorDomain` (number[2], optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square") {#colordomain}

* Default: `[0, 1]`

Controls how weight values are mapped to the `colorRange`, as an array of two numbers [`minValue`, `maxValue`].

> When `colorDomain` is specified, a pixel with `minValue` is assigned the first color in `colorRange`, a pixel with `maxValue` is assigned the last color in `colorRange`, and any value in between is linearly interpolated. Pixels with weight less than `minValue` gradually fade out by reducing alpha, until 100% transparency representing `0`. Pixels with weight more than `maxValue` are capped to the last color in `colorRange`.

Reviewer: > and any value in between is linearly interpolated.

Perhaps just "any value in between is interpolated"? I'm probably overthinking it, but I could see this implying interpolation in a linear color space, which the layer currently does not do.

> ... until 100% transparency representing `0`

Given signed colorDomain like `[-100, 100]`, I wouldn't be sure how to interpret this... do we need to say anything more about full transparency representing zero?

Author: The [description was wrong](https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/heatmap.ts#L105C13-L105C23), I've [corrected](https://github.com/visgl/deck.gl/pull/8909/commits/3f92c14f1115239d8adc07ae5ffdcf8114dff454) it
